### PR TITLE
2.6 rancher updates

### PR DESCRIPTION
**Pull request by KevinJoiner**

### CODE REVIEW

**Review by rmweir — CHANGES_REQUESTED** [review]
Can you remove 0402678ab57d4a6878ccf0a9c53fe38f5e66ae56 5a56a94923c367eba553824c9beebb59243dfae5 and their corresponding commits in the other two PRs? We do not want these changes for final release.

File: package/Dockerfile [hunk 1]
```diff
@@ -53,8 +53,8 @@ ENV CATTLE_SYSTEM_AGENT_UPGRADE_IMAGE rancher/system-agent:${CATTLE_SYSTEM_AGENT
 ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 100.0.3+up0.3.3
 
 # System charts minimal version
-ENV CATTLE_FLEET_MIN_VERSION=100.1.0+up0.4.0
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=1.0.6+up0.2.7
+ENV CATTLE_FLEET_MIN_VERSION=100.2.0+up0.5.0
```
Comment (macedogm) [inline package/Dockerfile:56]: Update the version following https://github.com/rancher/charts/pull/2346/files#diff-1d00b840e705f8e431bd805362d77aeda19db7c216857eac7bf94c28a0478d13R15

```suggestion
ENV CATTLE_FLEET_MIN_VERSION=100.2.0+up0.5.1
```

File: package/Dockerfile [hunk 2]
```diff
@@ -53,8 +53,8 @@ ENV CATTLE_SYSTEM_AGENT_UPGRADE_IMAGE rancher/system-agent:${CATTLE_SYSTEM_AGENT
 ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 100.0.3+up0.3.3
 
 # System charts minimal version
-ENV CATTLE_FLEET_MIN_VERSION=100.1.0+up0.4.0
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=1.0.6+up0.2.7
+ENV CATTLE_FLEET_MIN_VERSION=100.2.0+up0.5.1
+ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=1.0.7+up0.2.8
```
Comment (cbron) [inline package/Dockerfile:57]: confirmed both of these

**Review by macedogm — APPROVED** [review]

File: pkg/apis/go.mod [hunk 3]
```diff
@@ -8,11 +8,11 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/rancher/aks-operator v1.0.7
 	github.com/rancher/eks-operator v1.1.5
-	github.com/rancher/fleet/pkg/apis v0.0.0-20220916041858-eae202f68985
+	github.com/rancher/fleet/pkg/apis v0.0.0-20230123175930-d296259590be
 	github.com/rancher/gke-operator v1.1.4
 	github.com/rancher/norman v0.0.0-20220627222520-b74009fac3ff
 	github.com/rancher/rke v1.3.15
-	github.com/rancher/wrangler v1.0.1-0.20220520195731-8eeded9bae2a
+	github.com/rancher/wrangler v1.0.2
```
Comment (cbron) [inline pkg/apis/go.mod:15]: confirmed

**Review by cbron — APPROVED** [review]
lgtm

**Comment by rmweir** [comment]
Mentioned UI changes that don't have a corresponding release in ember and dashboard. Going to move forward for now.

**Review by rmweir — APPROVED** [review]

**Pull request merged**